ls8/cpu.c

```diff
@@ -69,35 +76,63 @@ void cpu_run(struct cpu *cpu)
     // `IR`, the _Instruction Register_
     unsigned int IR;
 
+    unsigned int operandA;
+    unsigned int operandB;
+
     int running = 1; // True until we get a HLT instruction
     while (running)
     {
-        // TODO
+        // Debugging/testing
+        printf("Registers:\n[ ");
```

Comment:
not to toot my own horn, but: great idea, will make debugging 10x easier

Reply:
If it ain't broke...